content/it/service-proxy.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Service Proxy
+status: Completed
+category: tecnologia
+tags: ["networking", "", ""]
+---
+
+## Cos'è
+
+Un service proxy intercetta il traffico verso o proveniente da un determinato [servizio](/it/service/), 
+vi applica delle logiche, e in seguito inoltra quel traffico a un altro servizio.
+Essenzialmente si comporta come un intermediario che raccoglie informazioni riguardo il traffico di rete e/o applica delle regole su di esso.
+
+## Quali problematiche affronta
+
+Per tenere traccia delle comunicazioni tra servizi, cioè traffico di rete, e potenzialmente trasformarlo o reindirizzarlo, bisogna raccogliere dati.
+Tradizionalmente, il codice che abilitava la raccolta di dati e la gestione del traffico di rete era incorporato all'interno di ogni applicazione.
+
+## In che modo aiuta
+
+Un service proxy permette di "esternalizzare" questa funzionalità.
+Quel codice non deve più necessariamente essere all'interno delle applicazioni,
+ma invece ora è incorporato all'interno del livello della piattaforma (cioè dove l'applicazione è in esecuzione).
+
+Fungendo da guardiani tra i servizi, i proxy forniscono informazioni riguardo a che tipo di comunicazione sta avvenendo.
```

Comment:
```suggestion
Fungendo da porta d'accesso ai servizi, i proxy forniscono informazioni riguardo a che tipo di comunicazione sta avvenendo.
```
premetto che non ho effettivamente mai sentito la traduzione di gatekeeper in questo contesto, ma mi pare giusto evidenziare che i service proxy qui hanno la funzione di "interceptors" più che di sicurezza (quest'ultima più resa dalla parola "guardiano"). Che ne pensi?

Reply:
beh, in effetti dopo si parla del fatto che possono rigettare una richiesta... non so, vedi tu, io preferisco "porta d'accesso" :D